eZ/Publish/Core/Search/Legacy/Content/Common/Gateway/CriterionHandler/FullText.php

```diff
@@ -243,6 +250,18 @@ public function handle(
                 )
             );
 
+        if (false === empty($languageSettings['languages'])) {
```

Comment:
Isn't it easier to read it using `!` ?

```suggestion
        if (f!empty($languageSettings['languages'])) {
```

Reply:
I prefer to use false instead of exclamation mark (and connect it with "Yoda style") for a few reasons, you can read about it here:
https://nvanesch.nl/blog/2018/06/12/why-i-hate-the-exclamation-mark-in-php/

But in this case, where return type is always boolean, I can change the style here, if you more like it :)